Makefile

```diff
@@ -73,7 +71,7 @@ default-config: install
 	bin/elastic-ingest --action config --service-type $(SERVICE_TYPE)
 
 docker-build:
-	docker build -t docker.elastic.co/enterprise-search/elastic-connectors:$(VERSION)-SNAPSHOT .
+	docker build -t docker.elastic.co/enterprise-search/elastic-connectors:$(VERSION)-SNAPSHOT -f app.Dockerfile .
```

Comment:
I've checked around and saw different suggestions:

1. Do `Dockerfile.{something}` - `Dockerfile.app`, `Dockerfile.ftest` (see https://stackoverflow.com/questions/62195202/multiple-dockerfiles)
2. Do `{something}.Dockerfile` - `app.Dockerfile`, `ftest.Dockerfile` (see https://stackoverflow.com/questions/27409761/docker-multiple-dockerfiles-in-project) - I did this cause of VSCode syntax highlighting
3. Do `Dockerfile_{something}` - `Dockerfile_app`, `Dockerfile_ftest` (see https://stackoverflow.com/questions/62195202/multiple-dockerfiles)

Overall there does not seem to be a standard way to do so so I decided to be very specific - `app.Dockerfile` looks a bit more verbose to me IMO. Additionally, we provide instructions everywhere about it.

I also feel it would be weird to have `Dockerfile` and `ftest.Dockerfile` together, but that's purely to my taste.

Do you feel we can rename app one to `Dockerfile` and leave the other one with prefix/suffix @seanstory?

Reply:
> Do you feel we can rename app one to Dockerfile and leave the other one with prefix/suffix @seanstory?

Yes, that's my loosely-held suggestion. I don't have the experience to say if that's the "standard" way to do things, but it feels more clear to me that way which one the customer wants, vs which is "sure, a dockerfile, but not what you're looking for."

Reply:
It's very common to have multiple docker files in projects. I personally got used to `Dockerfile.{something_extra}` e.g. there might be different docker files for different platforms or different environments. 

> it feels more clear to me that way which one the customer wants, vs which is "sure, a dockerfile, but not what you're looking for."

Part of me agrees with this so I think it's okay to leave `Dockerfile` as a main docker file and have an extra one which is called `ftest.Dockerfile`.